nbconvert/exporters/html.py

```diff
@@ -27,6 +29,58 @@
 from .templateexporter import TemplateExporter
 
 
+def find_lab_theme(theme_name):
+    """
+    Find a JupyterLab theme location by name.
+
+    Parameters
+    ----------
+    theme_name : str
+        The name of the labextension theme you want to find.
+
+    Raises
+    ------
+    ValueError
+        If the theme was not found, or if it was not specific enough.
+
+    Returns
+    -------
+    theme_name: str
+        Full theme name (with organization any, if any)
```

Comment:
Looks like a minor typo?

```suggestion
        Full theme name (with organization, if any)
```

Reply:
We could also say `scope` as an alternative to `organization`

Reply:
Thanks!